index.html

```diff
@@ -12,12 +12,138 @@
       <h1>Trivia Questions</h1>
     </header>
     <section class="centered">
+
+  
+      
       <form>
-        <button type="submit">Get New Questions</button>
+        <button type="button">Get New Questions</button>
       </form>
     </section>
     <main class="centered">
-      <!-- Your `.card` elements should show up here. -->
+
```

Comment:
It would probably be a lot easier to create the cards programmatically rather than manually creating templates for them.